parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -208,6 +212,7 @@ pub mod pallet {
 	{
 		fn on_initialize(_: BlockNumberFor<T>) -> Weight {
 			// Remove storage from previous block
+			PendingHighPriorityMessageCount::<T>::kill();
```

Comment:
The idea was to _decrement_ this counter after processing each high priority message.

We can't kill it every block, because there is no guarantee that all pending high priority messages will be processed every block.

Reply:
https://github.com/Snowfork/snowbridge/pull/949/commits/239f3c37a261c7bfda76fb6dcfbce5805eeecee4